layers/org/packages.el

```diff
@@ -388,3 +389,7 @@ Will work on both org-mode and any mode that accepts plain html."
 (defun org/init-htmlize ()
  (use-package htmlize
     :defer t))
+
+(defun org/init-orgit ()
+  (use-package orgit
+    :defer t))
```

Comment:
How the package will be loaded ? I guess you are missing a hook in a `:init` block.